### DISCUSSION TITLE
UPDATED for  where we check University BP Debate Tournaments

### DISCUSSION THREAD

**Discussion by isBlake** [Zaki-Mao/MUST-English-Debate-Team-EDT-#6] [Q&A] [answered]
UPCOMING Tournaments check https://home.worldschoolsdebatingleague.com/tournaments

**Comment by isBlake** [comment]
https://home.worldschoolsdebatingleague.com/tournaments

**Reply by Zaki-Mao** [reply]
https://wolce.org/asian-schools-debating-championship-3v3

**Comment by Zaki-Mao** [comment, marked as answer]
WUDC 2024 https://zh-cn.facebook.com/VietnamWUDC2024/

**Comment by Zaki-Mao** [comment]
https://www.facebook.com/story.php?story_fbid=204694325457664&id=100077512940660&sfnsn=mo
